MkDocs/monitor_csv_importer_8.6/docs/adjust_script.md

```diff
@@ -1,25 +1,144 @@
 # Objectives
 In this Exercise you will learn how to:
 
-* Adjust the required Node-RED script
-* Verify the script
+* Create the Monitor Device type
+* Inject and verify 1 row of data
+* Inject historical data from CSV file(s)
 
 ---
 *Before you begin:*  
 This Exercise requires that you:
 
 1. Have completed the pre-requisites required for [all labs](../prereqs)
 2. Have completed the [setup of Node-RED locally](../setup)
-3. Have [imported the Node-RED scripot](../get_script)
+3. Have [imported the Node-RED script](../get_script)
  
+
+!!! note
+    You need the credentials described in the [Pre-Requites](../prereqs) before proceeding with this exercise.
+
```

Comment:
Shouldn't a pre requisite also be the device type and devices for the pumps [Adjust_10.png](https://github.com/IBM/monitor-hands-on-lab/pull/52/commits/01bb1ef5f2e37b159a8b8f3077e2b70387085f00#diff-bf91717ed3c9b6f464ccb8660d989a73d41dd4f7d704d9807481a64b7e979b8d)

Reply:
No, as that is defined in the "Set Flow Data" function node and you adjust the device type in step 3.